native/dlaln2.go

```diff
@@ -0,0 +1,386 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import "math"
+
+// Dlaln2 solves a 1×1 or 2×2 linear system of equations of the form
```

Comment:
Yes, in the end it is just a careful division but the statement as it is is correct. Between two alternatives that I saw:
`Dlaln2 solves a linear system of na equations of the form ...`
`Dlaln2 solves a linear equation or a system of 2 equations of the form ...`.
I went with the second one.